spec/lib/video_info/providers/vkontakte_spec.rb

```diff
@@ -142,4 +143,32 @@
     its(:title)            { should eq 'SpaceGlasses are the future of computing' }
   end
 
+  context "with valid video and connection timeout" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.